samples/boards/nrf91/nrfx/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_GPIO=n
```

Comment:
Added a note in readme.

Reply:
I thought that you'd rather place this note here. Who reads documentation? ;-)

Reply:
sigh.

Reply:
@dbkinder I knew you would read it. :smile:
I was thinking about users who would take this sample as a starting point and modify it. I'm not sure all of them will look at README. But actually, this will be their mistake. :wink: